networkit/cpp/distance/test/DynSSSPGTest.cpp

```diff
@@ -63,6 +64,42 @@ TEST_F(DynSSSPGTest, testDynamicBFS_1edge) {
     });
 }
 
+TEST_F(DynSSSPGTest, testDynamicBFSEdgeDeletion) {
+/* Graph:
+    0    3   6
+     \  / \ /
+      2    5
+     /  \ / \
+    1    4   7
+ */
+    Graph G(8);
+
+    G.addEdge(0, 2);
+    G.addEdge(1, 2);
+    G.addEdge(2, 3);
+    G.addEdge(2, 4);
+    G.addEdge(3, 5);
+    G.addEdge(4, 5);
+    G.addEdge(5, 6);
+    G.addEdge(5, 7);
+
+    BFS bfs(G, 0);
+    bfs.run();
```

Comment:
Do we need to run `bfs` here? We never read the output before we run it again in line 96.